### ISSUE TITLE
Integration of Google's nearby share protocol

### ISSUE THREAD

**Issue by aayushrautela** [localsend/localsend#1122]
### Is your feature request related to a problem? Please describe.

Sharing files when direct connection isnt available

### Describe the solution you'd like

Below project implements lots of sharing protocol and can be integrated into localsend for beter usability.
https://github.com/vicr123/QNearbyShare

### Describe alternatives you've considered

KDE connect and snapdrop. kde connect being local whereas snapdrop not having proper integration with linux.

**Comment by Tienisto** [comment]
This is not feasible in the near future

**Comment by EtzBetz** [comment]
If it is done, it should be rather done for QuickShare, which is what NearbyShare has been switched to. QuickShare was developed by Samsung, but they teamed up with Google now.

**Comment by EtzBetz** [comment]
> If I am not wrong then samsung has adopted nearby share protocol for its quick share app. However, both the apps will now be called quick share. I think Google made a trade off when it comes to branding the app in exchange for Samsung to adopt it. 

Yes, I'm not entirely sure on that. I'm quite certain that at least Samsung has made some adjustments from their side on top, but they may be very well using most of Google's work :)

I think it would be useful if you'd change the title to call it QuickShare, as that's what it's called from now on ;)

**Comment by shockergit** [comment]
this is better than quick share. also i think it needs location info

**Comment by MrRulf** [comment]
This issue got closed, is localsend now quickshare compatible? 
I stumbled across https://github.com/Martichou/rquickshare which might be a good resource for this if it isn't yet.
I think rquickshare would also be a good foundation for the bluetooth discovery because then its directly compatible with many other devices - being a custom implementation the contacts only mode could be modified to support other sources for contacts aswell and not just google. 
Wifi direct isnt implemented yet but there aswell i think quickshare is a good foundation, it has encryption etc. And using it means a lot of devices compatible ootb
